client/components/button/docs/example.jsx

```diff
@@ -23,19 +25,38 @@ var Buttons = React.createClass( {
 	},
 
 	render: function() {
-		var toggleButtonsText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
+		var toggleText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
+		return config.isEnabled( 'devdocs/usage-stats' )
+			? this.renderNewStructure( toggleText )
```

Comment:
That "There are only two hard things in Computer Science: cache invalidation and naming things." :) Sure I will try to come up with better names.